pages/Signup/index.tsx

```diff
@@ -17,46 +17,47 @@ interface LoginField {
     password?: string;
 }
 
-const validate = (data: LoginField): ValidationError => {
-    const errors = microValidator.validate({
-        name: {
-            required: {
-                errorMsg: `Full Name is required`
-            }
-        },
-        username: {
-            required: {
-                errorMsg: `Username is required`
-            }
-        },
-        email: {
-            required: {
-                errorMsg: `Email is required`
-            },
-            email: {
-                errorMsg: 'Please enter a valid email'
-            }
-        },
-        password: {
-            required: {
-                errorMsg: `Password is required`
-            },
-            length: {
-                min: 6,
-                max: 12,
-                errorMsg: 'Password length between 6 and 12'
-            }
-        },
-    }, data)
-    
-    return errors
-}
-
 const Signup: React.FunctionComponent<RouteComponentProps> = ({
     history
 }: RouteComponentProps) => {
 
     const theme: AppTheme = useTheme();
+    const constants: AppConstants = useConstants();
+
+    const validate = (data: LoginField): ValidationError => {
```

Comment:
Instead of moving the whole function what I meant @yashkapoor8710 was this

```js
microValidator.validate(data, {
            username: {
                required: {
                    errorMsg: constants.loginValidation.username
                }
            },
            password: {
                required: {
                    errorMsg: constants.loginValidation.password
                },
                length: {
                    min: 6,
                    max: 12,
                    errorMsg: constants.loginValidation.passwordLength
                }
            },
        })
```